src/scripting/lua_game.cc

```diff
@@ -1235,7 +1251,7 @@ Objective
 
 .. class:: Objective
 
-   This represents an Objective, a goal for the player in the game. This is
+   This represents an Objective, a goal for the player in the scenario. This is
```

Comment:
well in fact objectives do not belong only to scenarios, but each wincondition sets an objective and a map can set an objective as well. so I believe game was correct here

Reply:
please add plural instead of singular ion line 1257 below : i.e. hide it from the users

mabe the same in line 1350 as well